src/index.js

```diff
@@ -77,13 +89,44 @@ export default ({
     ],
   }));
 
+  const remoteModuleBaseUrls = [];
   // support one-app-cli's "serve-module"
   // merge local with remote, with local taking preference
   oneAppDevCdn.get('/module-map.json', (req, response) => {
     settle([
       remoteModuleMapUrl
-        ? got(remoteModuleMapUrl).then(
-          r => JSON.parse(r.body),
+        ? got(remoteModuleMapUrl, {
+          agent: new ProxyAgent(),
+        }).then(
+          (r) => {
+            const remoteModuleMap = JSON.parse(r.body);
+
+            const { modules } = remoteModuleMap;
+            const oneAppDevStaticsAddress = `http://${req.headers.host}/static`;
+
+            Object.keys(modules).forEach((moduleName) => {
+              const module = modules[moduleName];
+
+              const parsedBundlePath = path.parse(module.node.url, '');
+
+              // store urls for later lookup when module requests are caught by one-app-dev-cdn
+              remoteModuleBaseUrls.push(
```

Comment:
is this list going to keep growing on every request for the module map ? might need to filter duplicates

Reply:
Yeah you are right. Fixed in 579b60c.